app/containers/message/Reply.tsx

```diff
@@ -96,21 +96,21 @@ interface IMessageReply {
 
 const Title = React.memo(
 	({ attachment, timeFormat, theme }: { attachment: IAttachment; timeFormat?: string; theme: TSupportedThemes }) => {
-		const time = attachment.message_link && attachment.ts ? moment(attachment.ts).format(timeFormat) : null;
-		return (
-			<View style={styles.authorContainer}>
-				{attachment.author_name ? (
-					<Text numberOfLines={1} style={[styles.author, { color: themes[theme].auxiliaryTintColor }]}>
-						{attachment.author_name}
-					</Text>
-				) : null}
-				{time ? <Text style={[messageStyles.time, { color: themes[theme].auxiliaryText }]}>{time}</Text> : null}
-				{attachment.title ? <Text style={[styles.title, { color: themes[theme].bodyText }]}>{attachment.title}</Text> : null}
-			</View>
-		);
+	  const time = attachment.message_link && attachment.ts ? moment(attachment.ts).format('YYYY-MM-DD HH:mm:ss') : null; // Modify the format here
```

Comment:
I disagree with approach.
It's going to display the full date every time and that takes a lot of space and creates clutter.

You could use a similar approach to RoomItem https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/e004ca39de022e8a2b5ef4f4680c302578f3483c/app/containers/RoomItem/index.tsx#L41, which formats the date relative to new Date()

Reply:
Please look through it now. I made changes as requested. Thank you for this idea!!